### ISSUE TITLE
Operation Rescue - Issue List

### ISSUE THREAD

**Issue by JJ173** [FAForever/faf-coop-maps#335]
Fixes marked with an `X` have been addressed, please feel free to report any bugs missed in the comment section below.

- [ ] Friendly Fire Issues with UEF Civilian Facility
- [ ] Potential Abuse in first objective (stuck units preventing expansion).
- [ ] Remove T3 Bots from Easy Difficulty
- [ ] Uneven Terrain ( Map used from stock vanilla so may be unfixable ).
- [ ] Fix truck waypoints when they leave the science facility.
- [ ] Add a timer for lower difficulties for the Experimental Spider attack during the escort.
- [ ] Increase AI Attacks during escort phase of the mission (Reported as too easy).
- [ ] Add objectives to ensure notifications about critical transport survival
- [ ] Scenario reported as stuck when T1 transports are destroyed.
- [ ] Scenario will freeze on a rare occasion after the patrols are destroyed (unable to fix, due to no errors in the console, but I will carry on investigating).
- [ ] Assign Cybran Commander Objective Earlier
- [ ] Restrict Quantum Gateway until trucks are evacuated
- [ ] Scenario doesn't end when a Commander is killed
- [ ] Remove all Cybran units when the Cybran ACU is terminated
- [ ] Investigate reported cinematic bug when killed by MLs (possibly related to the scenario not ending).
- [ ] Killing Jammer before scouting objective is complete causes scenario to freeze.

**Comment by TakesTheBiscuit** [comment]
Think i raised this issue in the wrong repo:
https://github.com/FAForever/fa/issues/4196

You've probably caught those issues?

**Comment by Garanas** [comment]
Yes, that was the wrong repository. I think I can transfer it, I'll try so tonight!

**Comment by Garanas** [comment]
Transferred it, see also: https://github.com/FAForever/faf-coop-maps/issues/384
